x-pack/test/api_integration/deployment_agnostic/README.md

```diff
@@ -117,25 +117,34 @@ test example
 ```ts
 export default function ({ getService }: DeploymentAgnosticFtrProviderContext) {
   const roleScopedSupertest = getService('roleScopedSupertest');
-  let supertestWithAdminScope: SupertestWithRoleScopeType;
+  let supertestViewerWithApiKey: SupertestWithRoleScopeType;
+  let supertestEditorWithCookieCredentials: SupertestWithRoleScopeType;
 
-  describe('compression', () => {
+  describe('test suite', () => {
     before(async () => {
-      supertestWithAdminScope = await roleScopedSupertest.getSupertestWithRoleScope('admin', {
+      supertestViewerWithApiKey = await roleScopedSupertest.getSupertestWithRoleScope('viewer', {
         withInternalHeaders: true,
         withCustomHeaders: { 'accept-encoding': 'gzip' },
       });
+      supertestEditorWithCookieCredentials = await roleScopedSupertest.getSupertestWithRoleScope('editor', {
+        withInternalHeaders: true,
+        useCookieHeader: true,
+      });
     });
     after(async () => {
       // always invalidate API key for the scoped role in the end
-      await supertestWithAdminScope.destroy();
+      await supertestViewerWithApiKey.destroy();
```

Comment:
Nit: even though it's only a README, the newly added `supertestEditorWithCookieCredentials` should also be cleaned up.

Reply:
Actually it is not needed, but I admit that is not obvious. `supertestEditorWithCookieCredentials` uses cached Cookie generated with `samlAuth` service and we don't expose Cookie session cleanup (only force reset, but that is a different use case).

I will add explicit comment about it

Reply:
Ah, okay. Well, that can be a bit confusing then. Thanks for adding a comment to clarify it.